addons/settings/script_component.hpp

```diff
@@ -138,6 +139,12 @@
     (uiNamespace getVariable QGVAR(serverTemp))  getVariable [setting, [[setting,  "server"] call FUNC(get), nil]] select 0\
 ] select (["client", "mission", "server"] find TEMP_PRIORITY(setting)))
 
+#define SOURCE_VALUE(setting,source) ([\
```

Comment:
TEMP_VALUE_SOURCE would be a more accurate name.
Also, the macro above could then be simplified:
```cpp
#define TEMP_VALUE(setting) TEMP_VALUE_SOURCE(setting,TEMP_PRIORITY(setting))
```